server/test/controllers/admin/AdminImportControllerTest.java

```diff
@@ -1937,7 +1937,7 @@ public void hxSaveProgram_noDuplicatesEnabled_addsAnEmptyStatus() {
               },
               "id" : 3,
               "universal" : true,
-              "primaryApplicantInfoTags" : []
+              "primaryApplicantInfoTags" : ["APPLICANT_DOB"]
```

Comment:
should match tags in here: https://github.com/civiform/civiform/blob/c06128569a5079febaa5e7c029a4c7b9c88ba8f2/server/app/services/question/PrimaryApplicantInfoTag.java#L15